src/main/java/org/snomed/snowstorm/core/rf2/rf2import/ImportService.java

```diff
@@ -25,7 +36,9 @@
 @Service
 public class ImportService {
```

Comment:
Nice refactoring of this class although be wary of repeating lengthy java doc descriptions when it's a private method anyway. A single comment somewhere may be cleaner?

Reply:
Fixed